### ISSUE TITLE
Why the same setup can look so different?

### ISSUE THREAD

**Issue by budimanjojo** [romkatv/powerlevel10k#1534]
Hello, I don't know if I should ask here because I don't think this is powerlevel10k issue. But I don't know where else should I go. I've been using Powerlevel10k like for the last 3 years or something. And recently I see there are a lot of new options worth trying out so I did a complete `configure p10k` and start from there. I have everything configured and looking so good so I happily sync my dotfiles across my other machine. And on the other machine I see some glyphs are not working at all, and here they are:
```
    typeset -g POWERLEVEL9K_PROMPT_CHAR_{OK,ERROR}_VIINS_CONTENT_EXPANSION='⮞'
    typeset -g POWERLEVEL9K_PROMPT_CHAR_{OK,ERROR}_VICMD_CONTENT_EXPANSION='⮞'
    typeset -g POWERLEVEL9K_PROMPT_CHAR_{OK,ERROR}_VIVIS_CONTENT_EXPANSION='⮟'
    typeset -g POWERLEVEL9K_PROMPT_CHAR_{OK,ERROR}_VIOWR_CONTENT_EXPANSION='⮞'
```
The icons above are displayed as a square in the terminal.
I have the same terminal and it's config file (alacritty), same font (UbuntuMono Nerd Font), same zshrc. The difference is the machine that work is using Ubuntu 20.04 with Unity launcher and the other machine is running Archlinux with i3wm.

I have nerd-fonts complete installedon both machines, The machine that work can display those icons using any nerd font. And the other machine can't display them at all regarding of nerd font I tried. 

Once again I apologize for asking this here because this is not caused by p10k at all but I hope somebody here can help me. Thanks before :smile:

**Comment by romkatv** [comment]
If a terminal cannot display `⮞`, it can be an issue with the font, the terminal, or the WM. This isn't specific to powerlevel10k. You simply copy-paste the symbol into your terminal (e.g. into `cat`) and you'll see that it doesn't render. You can also run `echo ⮞`.

**Comment by romkatv** [comment]
I would first check whether UbuntuMono Nerd Font actually has this symbol (U+2B9E). You probably have some kind of character map application that should allow you to do this.

If the font is missing this glyph, on some systems the glyph can still be displayed via fallbacks.

**Comment by budimanjojo** [comment]
Yes, it can't display it at all. I tried using echo and it display the same square symbol.
As for the font and terminal are the same, that means the problem is with the WM?
How do I check whether UbuntuMono Nerd Font has this symbol? And it's not just UbuntuMono. Like I said before, the machine that works can display that symbol using any Nerd Font, but the other machine can't display it at all using any font (I tried like 20 different Nerd Fonts).

Thank you so much for answering me!! :smile:
